src/dotnet/Chat.UI.Blazor/Pages/ChatPage.razor

```diff
@@ -31,7 +45,9 @@
 </RenderIntoStack>
 
 <RenderIntoSlot Name="MainRightDrawer">
-    @if (chatContext.HasChat) {
+    @if (ReferenceEquals(chat, SpecialChat.Unavailable) && m.IsInaccessiblePlace) {
```

Comment:
I don't like these conditions. I think it would be better if we pass state via ChatContext and then apply appropriate rendering insine ChatRightPanel component.

Reply:
We need to discuss this. I had to make skeletons for inaccessible place.